### PR TITLE
Original project-flow file converted to md

### PR DESCRIPTION
**Pull request by zilva149**
Adding project-flow.md

### CODE REVIEW

File: project-flow.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Projekto eiga
+
+## Bendrinis projekto aptarimas:
```
Comment (EvgenySivaykin) [inline project-flow.md:3]: Uppercase, this. Please

File: project-flow.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- Frontend projektas su pavyzdiniais duomenimis, kuriuos imsime iš susikurtų JSON failų asinchroniško JavaScript’o būdu (fetch(), async, await).
+- Bendravimas Github’e atliekamas anglų kalba, Discord’e lietuvių.
+- Užduočių aptarimas bus vykdomas gyvai.
```
Comment (zilva149) [inline project-flow.md:7]: trablaba

**Review by EvgenySivaykin — DISMISSED** [review]

File: project-flow.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        - Ar galima kur struktūroje naudoti tinkamesnį semantinį tagą?
+        - Patikrinti kaip atrodo responsyvus dizainas skirtinguose ekrano dydžiuose.
+        - Ar selektoriai yra rašomi tvarkingai struktūros eiliškumo tvarka?
+        - Ar atlikau visas užduotis?
+    - Dar kartą susinchronizuoti savo branch’ą su develop branch’u.
+    - Jei nekyla jokių konfliktų, daryti Pull Request’ą.
+
+## Kodo peržiūra Pull Request’o metu:
+
+- Nedarome skubotų patvirtinimų.
```
Comment (Rasosm) [inline project-flow.md:41]: Good point

**Review by liepsneletech — CHANGES_REQUESTED** [review]
Why?

File: project-flow.md (hunk 1 above)
Comment (zilva149) [reply project-flow.md:3]: no

File: project-flow.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Frontend projektas su pavyzdiniais duomenimis, kuriuos imsime iš susikurtų JSON failų asinchroniško JavaScript’o būdu (fetch(), async, await).
+- Bendravimas Github’e atliekamas anglų kalba, Discord’e lietuvių.
+- Užduočių aptarimas bus vykdomas gyvai.
+
+## Užduoties atlikimas:
+
+- ### Nuo ko pradėti:
+    - Perskaičius atliekamą užduotį visuomet iš anksto pasiteirauti dėl kylančių klausimų. Užduotis bus galima detaliau papildyti.
+    - Prieš darbą nusikopijuoti repozitoriją `git clone <adresas>` komanda terminale arba per vscode source control’ą.
+    - Pradėjus atlikinėti užduotį, susikurti feature-branch’ą ir savą užduotį atlikinėti ten.
```
Comment (liepsneletech) [inline project-flow.md:14]: Blabla

**Review by liepsneletech — CHANGES_REQUESTED** [review]
Bad

File: project-flow.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zilva149) [reply project-flow.md:7]: try harder

**Review by liepsneletech — APPROVED** [review]
After changes it looks fantastic!

**Review by Dorenus — APPROVED** [review]
Well done, nothing to add really

**Review by EvgenySivaykin — APPROVED** [review]

**Review by Rasosm — APPROVED** [review]

**Pull request merged**